taxi/forms.py

```diff
@@ -50,3 +50,15 @@ def validate_license_number(
         raise ValidationError("Last 5 characters should be digits")
 
     return license_number
```

Comment:
Code Quality Issue: The `validate_license_number` function is not used anywhere else in this file, but it is defined globally. It is a good practice to define functions as close as possible to their point of use. If this function is not used in other files, consider moving it inside the class where it is used.